sematic/config.py

```diff
@@ -107,6 +107,11 @@ def server_url_from_env_vars(self):
                 f"Cannot construct server URL from env vars if "
                 f"{SEMATIC_SERVER_ADDRESS_ENV_VAR} is not set."
             )
+        if server_address is not None and (
+            server_address.startswith("http://")
+            or server_address.startswith("https://")
+        ):
```

Comment:
Why this change?

Reply:
A few reasons. One is because we didn't provide any way to use https here otherwise. Another is that we had one case where an env var is expected to set the server address without the http (`SEMATIC_SERVER_ADDRESS`) and one where it is expected to have it (`SEMATIC_API_ADDRESS`). This makes it so `SEMATIC_WORKER_API_ADDRESS` works both ways. We should make it more consistent ideally; filed a ticket, #126.